frontend/src/turbo/flash-stream-action.ts

```diff
@@ -0,0 +1,15 @@
+import { StreamActions, StreamElement } from '@hotwired/turbo';
+
+export function registerFlashStreamAction() {
+  StreamActions.flash = function dialogStreamAction(this:StreamElement) {
+    const content = this.templateElement.content;
+
+    const flash = content.querySelector('.flash') as HTMLElement;
+    if (flash.dataset.singleton) {
+      const elements = document.querySelectorAll('.flash');
+      elements.forEach((element) => element.parentElement?.remove());
+    }
```

Comment:
What does this do?

Reply:
We thought about adding some additional logic to replace existing flashes, but that ended up not being necessary. I'll remove it